File: trabalho 1/VitorBasso11611BCC034/programas tarefa 1/java/micro/micro11.java
public class micro11{
    public static void main(String[] args){
        int numero, x;
        System.out.print("Digite um numero: ");
        numero = Integer.parseInt(System.console().readLine());
        x = verifica(numero);
        if(x == 1){
            System.out.println("Numero positivo");
        }else if(x == 0){
            System.out.println("Zero");
        }else{
            System.out.println("Numero negativo");
        }
    }
    public static int verifica(int n){
        int res;
        if(n > 0){
            res = 1;
        }else if (n < 0){
            res = -1;
        }else{
            res = 0;
        }
        return res;
    }
}

File: trabalho 5/codigos/lua/micro/micro11.lua
function int main()
    int numero, x
    print("Digite um numero ")
    numero = io.read('*n')
    x = verifica(numero)
    if x == 1 then
        print("Numero positivo\n")
    else if x == 0 then
        print("Zero\n")
    else
        print("Numero negativo\n")
    end
    end
    return 1
end

function int verifica(int n)
    int res
    if n > 0 then
        res = 1
    else if n < 0 then
        res = -1
    else
        res = 0
    end
    end
    return res
end

main()
